tidb-cloud/import-with-mysql-cli.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Import Data into TiDB Cloud via MySQL Commands in Terminal
+summary: Learn how to import Data into TiDB Cloud via MySQL CLI.
+---
+
+# Import MySQL Data via MySQL Commands in Terminal
+
+## Prerequisites:
+
+Before you can import MySQL data to TiDB Cloud, you should have the following prerequisites in place:
+
+- Access to your TiDB Cloud Cluster.
+- Necessary connection details obtained from the TiDB Cloud portal.
+- An initial password created for your TiDB user (if not already done).
+- TiDB cluster connection information.
+
+## Steps:
+
+### 1. Connect to your TiDB Cloud Cluster
+
+1. Access the TiDB Cloud portal and obtain the necessary connection details.
+2. Create an initial password for your TiDB cluster if you haven't already.
+3. Select your connect options: Choose 'Endpoint Type' as 'Public', select 'Connect With' as 'MySQL CLI' and choose the operating system you are using. Obtain the cluster connection information.
+
+### 2. Install MySQL Command Line Client
+
+You need to have MySQL Command Line Client installed on your local machine. You can download it from the official MySQL website if it's not already installed.
+
+### 3. Define Table and Insert Sample Data
+
+Before importing data, you need to prepare the table structure and insert real sample data into it. Here's an example using a hypothetical "products" table:
+
+{{< copyable "sql" >}}
+```sql
+-- Create a table in your TiDB database
+CREATE TABLE products (
+    product_id INT PRIMARY KEY,
+    product_name VARCHAR(255),
+    price DECIMAL(10, 2)
+);
+
+-- Insert sample data into the table
+INSERT INTO products (product_id, product_name, price) VALUES
+    (1, 'Laptop', 999.99),
+    (2, 'Smartphone', 499.99),
+    (3, 'Tablet', 299.99);
+```
+
+### 4. Import Data from SQL File
+
+1. Provide a real SQL file (e.g., `product_data.sql`) that contains the data you want to import. This SQL file should contain INSERT statements with real data.
```

Comment:
Does the `product_data.sql` refer to the SQL file above? Then we can name this file a little bit earlier.